packages/next/src/lib/metadata/resolve-metadata.ts

```diff
@@ -278,31 +313,23 @@ export async function accumulateMetadata(
         generatedMetadata instanceof Promise
           ? await generatedMetadata
           : generatedMetadata
-    } else {
+    } else if (metadataExport !== null && typeof metadataExport === 'object') {
+      // This metadataExport is the object form
```

Comment:
If there is no metadata we should continue. Probably don’t want the merge function to even have to check that metadata is not null

Reply:
we can't continue since there's might be static metadata needs to be merged into resolved metadata, that will be handled in `merge()` below

Reply:
Ah my bad